### PR TITLE
TW 339: development yaml link broken

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
Removes a broken link (since the topic would've linked to several files, and an example is given in the text anyway).

### CODE REVIEW

File: docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ If you operate a Temporal Cluster using our [Helm charts](https://github.com/tem
 **Persistence configuration**
 
 1. Add the `advancedVisibilityStore: es-visibility` key-value pair to the `persistence` section.
-   The [development_es.yaml](https://github.com/temporalio/temporal/blob/master/config/development_es.yaml) file in the `temporalio/temporal` repo is a working example.
+   There are several development_es.yaml files in the `temporalio/temporal` repo.
```
Comment (flossypurse) [inline docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md:34]:
```suggestion
   There are several development_es.yaml files in the `temporalio/temporal` repo you can look at for example usage.
```

Comment (flossypurse) [reply docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md:34]: Maybe just link to the root of the repo?

File: docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ If you operate a Temporal Cluster using our [Helm charts](https://github.com/tem
 **Persistence configuration**
 
 1. Add the `advancedVisibilityStore: es-visibility` key-value pair to the `persistence` section.
-   The [development_es.yaml](https://github.com/temporalio/temporal/blob/master/config/development_es.yaml) file in the `temporalio/temporal` repo is a working example.
+   [There are several development_es.yaml files](https://github.com/temporalio/temporal/tree/master/config) in the `temporalio/temporal` repo you can look at for example usage.
```
Comment (djmagee) [inline docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md:34]:
```suggestion
   For example usage, you can look at [several development_es.yaml files](https://github.com/temporalio/temporal/tree/master/config) in the `temporalio/temporal` repo.
```
These edits place the goal before the action per https://developers.google.com/style/sentence-structure. The changes also conveniently avoid the construction "There are" (which is what initially caught my eye).

**Comment by DJSanti** [comment]
@djmagee yarn gen was run. are we good?

**Review by djmagee — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**